web/pgadmin/browser/server_groups/servers/__init__.py

```diff
@@ -1763,7 +1763,7 @@ def reload_configuration(self, gid, sid):
 
         if conn.connected():
             # Execute the command for reload configuration for the server
-            status, rid = conn.execute_scalar("SELECT pg_reload_conf();")
+            status, _ = conn.execute_scalar("SELECT pg_reload_conf();")
```

Comment:
execute_void can be used if SQL is returning nothing.

Reply:
The above code is very old, I have fixed the SonarQube code smell. Leaving it for now.